paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/CloneActionFactory.java

```diff
@@ -68,31 +68,31 @@ public void printHelp() {
 
         System.out.println("Syntax:");
         System.out.println(
-                "  clone --warehouse <warehouse_path> "
-                        + "[--database <database_name>] "
-                        + "[--table <table_name>] "
-                        + "[--catalog_conf <source-paimon-catalog-conf> [--catalog_conf <source-paimon-catalog-conf> ...]] "
-                        + "--target_warehouse <target_warehouse_path> "
-                        + "[--target_database <target_database_name>] "
-                        + "[--target_table <target_table_name>] "
-                        + "[--target_catalog_conf <target-paimon-catalog-conf> [--target_catalog_conf <target-paimon-catalog-conf> ...]] "
+                "  clone --warehouse <warehouse_path> \n"
+                        + "[--database <database_name>] \n"
+                        + "[--table <table_name>] \n"
+                        + "[--catalog_conf <source-paimon-catalog-conf> [--catalog_conf <source-paimon-catalog-conf> ...]] \n"
+                        + "--target_warehouse <target_warehouse_path> \n"
+                        + "[--target_database <target_database_name>] \n"
+                        + "[--target_table <target_table_name>] \n"
+                        + "[--target_catalog_conf <target-paimon-catalog-conf> [--target_catalog_conf <target-paimon-catalog-conf> ...]] \n"
```

Comment:
I think there is a problem in this format: actually a flink run command should be in 'one line', otherwise we should use a '\' to break line (see https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/deployment/cli/). I think we should add the '\' if we want to break lines for more readable message.

Reply:
Thanks for your remind!Got it.